modules/apps/commerce/headless/headless-commerce/headless-commerce-admin-shipment-impl/src/main/java/com/liferay/headless/commerce/admin/shipment/internal/dto/v1_0/converter/ShipmentDTOConverter.java

```diff
@@ -0,0 +1,73 @@
+/**
```

Comment:
When introducing a new class that implements an interface
        take into account:

* The class name should end with the interface name
* The package of the class matches the package of the interface



Refer to documentation: